README.md

```diff
@@ -587,6 +587,10 @@ It stars backend service with embedded bolt store on port `8080` with basic auth
 
 ### Frontend development
 
+#### Developer guide
+
+Frontend guide can be found here: [./frontend/Readme.md](./frontend/Readme.md)
```

Comment:
Moved it above because it is good to read first "how" instead of "what" you should do